test/unit/vdom/builder.js

```diff
@@ -79,5 +79,32 @@ describe('create()', () => {
         },
       }))());
     });
+
+    it('should allow arrays as children', (done) => {
+      fixture(new (define('x-test', {
+        render() {
+          return (
+            h('div',
+              h('span', { id: 1 }),
+              [h('span', { id: 2 }), h('span', { id: 3 })],
+              [h('span', { id: 4 }), h('span', { id: 5 }), h('span', { id: 6 })]
+            )
+          );
+        },
+        rendered({ shadowRoot }) {
+          expect(shadowRoot.innerHTML).to.equal(
+            '<div>' +
+              '<span id="1"></span>' +
+              '<span id="2"></span>' +
+              '<span id="3"></span>' +
+              '<span id="4"></span>' +
+              '<span id="5"></span>' +
+              '<span id="6"></span>' +
+            '</div>'
```

Comment:
Then the whitespace in the literal would fail the test (vdom doesn't generate whitespace).